lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -278,4 +269,11 @@ private APIGatewayProxyResponseEvent handleException(
     private APIGatewayProxyResponseEvent createNoContentResponse() {
         return new APIGatewayProxyResponseEvent().withStatusCode(HttpStatusCode.NO_CONTENT);
     }
+
+    private void initialServiceRegistry() {
+        this.serviceRegistry.loadKeyStore();
+        this.kbvStorageService = serviceRegistry.getKBVStorageService();
+        this.sessionService = serviceRegistry.getSessionService();
+        this.auditService = serviceRegistry.getAuditService();
```

Comment:
perhaps we could set this field in the constructor as the event queue url change will require a redeploy anyway

Reply:
This method can be removed
The following lines can be all be moved to the constructor:
this.kbvStorageService = serviceRegistry.getKBVStorageService();
this.sessionService = serviceRegistry.getSessionService();
this.auditService = serviceRegistry.getAuditService();

The keystore loading can just happen in the KBVGatewayFactory as-is
If we have a new KBVQuestion provider (which is looking likely) the keystore loading cannot remain here anyway